manager/manager.py

```diff
@@ -570,6 +571,41 @@ def generate_dashboard(self):
                 str(self.jobs_total) + '</td></tr>',
                 'html.parser'))
 
+            # build job failures table
+
+            jobFailures = self.get_job_failures(totalJobs)
+            body.append(BeautifulSoup('<h2>Failed Jobs</h2>', 'html.parser'))
+            failureTable = BeautifulSoup('<table id="failed" cellpadding="4" border="1" style="border-collapse:collapse"></table>','html.parser')
+            failureTable.table.append(BeautifulSoup(
+                '<tr id="header"><th class="hdr">Time</th><th class="hdr">Errors</th><th class="hdr">Source</th><th class="hdr">Destination</th><th class="hdr">Job ID</th></tr>',
+                'html.parser'))
+
+            for i in range(0, MAX_FAILURES):
+                if i >= len(jobFailures):
+                    break
+
+                item = jobFailures[i]
+
+                identifier = item['identifier']
+                identifiers = identifier.split("/")
+                sourceSubPath = "/".join(identifiers[:2])
+                gogs_url = self.gogs_url
+                if gogs_url == None :
+                    gogs_url = 'https://git.door43.org'
+
+                sourceUrl = gogs_url + "/" + sourceSubPath
+                destinationUrl = 'https://live.door43.org/u/' + identifier
```

Comment:
Example:
        if 'cdn_bucket' in event:
            # this i
            cdn_bucket = event['cdn_bucket']
            prefix = cdn_bucket[:-(len('cdn.door43.org'))]
            destination_url = '{0}door43.org'.format(prefix)